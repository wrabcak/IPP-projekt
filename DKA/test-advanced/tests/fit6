(
{s, p, q, r ,nonTermState, fin1},
{'a', '''', '{', ')', 'b', 'č' ,'b' }, 
{
s 'č' -> p,
q 'a' -> r, 
q '{' -> r, 
r 'a' -> r,
p '''' -> q,
r '' -> s
},
r, 
{fin1, s, r} 
)
