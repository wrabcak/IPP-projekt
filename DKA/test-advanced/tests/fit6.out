(
{p, q, r},
{'''', ')', 'a', 'b', '{', 'č'},
{
p '''' -> q,
q 'a' -> r,
q '{' -> r,
r 'a' -> r,
r 'č' -> p
},
r,
{r}
)